lib/streamlit/runtime/state/__init__.py

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 from streamlit.runtime.state.common import WidgetArgs, WidgetCallback, WidgetKwargs
+from streamlit.runtime.state.query_params import QueryParams
```

Comment:
Do we ever import `QueryParams` from `streamlit.runtime.state`? I couldn't find any instances of us doing so in subsequent PRs, so we should just get rid of this reexport if that's the case